lib/instance_billing_flavor_check/utils.py

```diff
@@ -139,6 +141,42 @@ def get_rmt_ip_addr():
     logger.info('Could not determine update server IP address')
 
 
+def _get_proxies():
+    """
+    Get proxy info.
+
+    If any proxy has been set up,
+    this method returns the proxy info in a dictionary.
+
+    Otherwise, returns None
+    """
+    if os.environ.get('http_proxy') or os.environ.get('https_proxy'):
+        logger.info('Using proxy settings present on the environment.')
+        # by default/if None requests relies on the environment variables
+        # HTTP_PROXY and HTTPS_PROXY
+        return None
+
+    proxies = {}
+    proxy_config = []
+    try:
+        with open(PROXY_CONFIG_PATH, 'r') as proxy_file:
+            proxy_config = proxy_file.readlines()
+    except FileNotFoundError:
+        logging.info('No proxy config file found on %s', PROXY_CONFIG_PATH)
```

Comment:
The user is not telling us to use a proxy, it is a discovery process, as such there is no need to tell the user anything about our discovery process. `pass` or `return`